### PR TITLE
OCT-1471 & OCT-1541 (client) Rewards calculator: update to improve results

### PR DESCRIPTION
**Pull request by jmikolajczyk**

### CODE REVIEW

**Comment by aziolek** [comment]
@jmikolajczyk: we merged the release into the `master`. You can rebase to `master`.

File: backend/app/exceptions.py [hunk 1]
```diff
@@ -272,3 +272,11 @@ class InvalidMatchedRewardsStrategy(OctantException):
 
     def __init__(self):
         super().__init__(self.description, self.code)
+
+
+class EstimateRewardsNotSupported(OctantException):
+    code = 500
```
Comment (aziolek) [inline backend/app/exceptions.py:278]: @kgarbacinski: why `500`? 500 means unexpected. It's not unexpected. I suggest 404, maybe.

File: client/src/components/Earn/EarnRewardsCalculator/EarnRewardsCalculator.tsx [hunk 2]
```diff
@@ -43,20 +40,23 @@ const EarnRewardsCalculator: FC = () => {
 
   // eslint-disable-next-line react-hooks/exhaustive-deps
   const fetchEstimatedRewardsDebounced = useCallback(
-    debounce(({ amountGlm, numberOfDays }) => {
+    debounce(({ amountGlm, numberOfEpochs }) => {
       const amountGlmWEI = formatUnitsBigInt(parseUnitsBigInt(amountGlm, 'ether'), 'wei');
-      const numberOfDaysNumber = parseInt(numberOfDays, 10);
+      // const numberOfDaysNumber = parseInt(numberOfDays, 10);
```
Comment (aziolek) [inline client/src/components/Earn/EarnRewardsCalculator/EarnRewardsCalculator.tsx:45]: Can be removed.

**Review by aziolek — CHANGES_REQUESTED** [review]
I will add the rest of the review after testing the feature in PR environment.

File: backend/app/exceptions.py (hunk 1 above)
Comment (kgarbacinski) [reply backend/app/exceptions.py]: it must've slipped through the be's code review, just removed it and updated your branch

**Review by aziolek — CHANGES_REQUESTED** [review]
@jmikolajczyk, @kgarbacinski: I deployed this PR to PR environment [here](https://pr-140-client.octant.wildland.dev/). I want to know how much ETH and MF 5000 GLM will generate if I lock it for the next 270 days. According to the app, 0. Please fix it before tests can proceed. Or is it calculator can't work in PR environment (why?) and this needs to be checked on UAT / MT?

image

image

**Review by aziolek — APPROVED** [review]

**Pull request merged**